Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yastdevel/cpp
+RUN zypper --gpg-auto-import-keys --non-interactive in --no-recommends \
+  "pkgconfig(gio-unix-2.0)" \
+  "pkgconfig(glib-2.0) >= 2.31.0" \
+  "pkgconfig(gtk+-3.0)" \
+  "pkgconfig(libgnome-menu-3.0)"
```

Comment:
The *"install buildrequires for this spec file"* thingy would be quite tricky:
- You would need additional magic (script) for that, having a static `Dockerfile` makes it much simpler and understandable.
- `zypper install` refreshes the repositories although everything is already installed, full refresh of all repositories takes quite long time (~30 seconds?), this would slow down every build.
- The dependencies usually change quite rarely, adding it manually into one more file is acceptable to me.